### ISSUE TITLE
Enhancement : Allow users to store their own Jython functions

### ISSUE THREAD

**Issue by ettorerizza** [OpenRefine/OpenRefine#1232]
I often reuse the same custom functions when I work in Open Refine with Jython. I ended up storing them all in a module, which I call if necessary. But it's not very user friendly. 

![screenshot-127 0 0 1-3333-2017-08-12-15-22-23](https://user-images.githubusercontent.com/6286410/29241117-570dbd3e-7f73-11e7-8d4d-9c0ca8752ddf.jpeg)

As you can see, three lines of boilerplates are needed for an oneliner. Would not it be a good idea to allow users to store their own Jython functions in a more flexible way? In this way, it would even be possible to share them between users! :)

**Comment by thadguidry** [comment]
You just need to custom package them so Jython and Butterfly can see them.
Just put your packages in Jython folder. You can ask Jython folks more how
to do this. Then when our Butterfly inits Jython your set.

**Comment by thadguidry** [comment]
@ettorerizza Can we close this ?  or do you need something more from us on this issue ?

**Comment by ettorerizza** [comment]
@thadguidry Yes, of course, you can close this thread if you want. I keep in mind the idea that GREL or Python code snippets exchangeable between users would be a good idea, but it's not a priority.

**Comment by thadguidry** [comment]
@ettorerizza Ah, you want a way to share Expressions...like perhaps having a window in OpenRefine to search by categories and then scrolling in that category to see the community Expressions, etc. and then you can click on one to use it.  Do you ever use the "Starred" feature ?  Anyways, similar to our own Starred but allow a more flexible, larger, community sharing window for searching for Expressions.  OK understood.  Let's keep this open and rename the title to reflect the exact need.

**Comment by wetneb** [comment]
More discussion of this in #912, just closed as a duplicate of this.
